### PR TITLE
New branch for changes

### PR DESCRIPTION
**Pull request by Kerimli98**
I added some **changes**

- change1
- change2

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import WatchList from "./components/WatchList";
 function App() {
   return (
     <div className="App">
+      {/* first change */}
```
Comment (amatanat) [inline src/App.js:9]: Please revert this change. We don't need this.

File: src/index.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
   <React.StrictMode>
     <BrowserRouter>
+      {/* second change */}
```
Comment (amatanat) [inline src/index.js:12]: 👍

**Review by amatanat — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (Kerimli98) [reply src/App.js:9]: Okay will update asap

Comment (Kerimli98) [reply src/App.js]: Updated here [3b15139](https://github.com/Kerimli98/studioGhibli/pull/4/commits/3b15139ce2d6073e4d25a6f397f7cef26d80d335)

**Review by amatanat — APPROVED** [review]
LGTM.
